cdap-docs/cdap-apps/source/hydrator/custom.rst

```diff
@@ -449,3 +449,255 @@ Plugin Deployment
 .. include:: ../../../developers-manual/source/building-blocks/plugins.rst 
    :start-after: .. _plugins-deployment-artifact:
    :end-before:  .. _plugins-use-case:
+
+Configuration JSON for a plugin (for Webapp UI alone)
+======================================================
+Every property of a plugin is represented, by default, as input fields in the UI. This can be however changed to specific widgets that closely match the type of the property in a plugin. This can be specified as JSON for the UI to consume.
```

Comment:
can you wrap lines to 120 chars across this PR? makes it easier to review, even in future.